tests/Unit/Controllers/ExecutiveControllerTest.php

```
@@ -100,13 +100,36 @@ public function testDashboardUsesOrgId(): void
         $this->assertEquals(10, $this->user['org_id']);
     }
 
+    public function testDashboardSkipsQueriesForRemovedTemplateSections(): void
+    {
+        $sqls = [];
+        $db = $this->createMock(\StratFlow\Core\Database::class);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)